<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>638</width>
    <height>535</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Analizador</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="boton_buscarImagen">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>20</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Buscar</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_imagenCargada">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>531</width>
      <height>371</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>Imagen no cargada</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="text_RutaImagen">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>491</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Imagen:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_evaluarFruto">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>90</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Evaluar fruto</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="caja_resultado">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>450</y>
      <width>531</width>
      <height>61</height>
     </rect>
    </property>
    <property name="plainText">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>430</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>RESULTADO DE EVALUACIÃ“N:</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
